components/molecules/Experiment.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+/**
+ * Displays an experiment card on the page
+ */
+
+export const Experiment = (props) => {
+  const tagColours = {
+    active: "custom-blue-experiment-blue",
+    coming_soon: "gray-experiment",
+    alpha: "custom-blue-experiment-blue",
+  };
+  return (
+    <div
+      className={`shadow-experiment-shadow p-4 border-b-4 h-400px ${
+        "border-" + tagColours[props.tag] || "bg-gray-experiment"
+      }`}
+      data-testid={props.dataTestId}
+      data-cy={props.dataCy}
+    >
+      <h2 className="mb-2 text-p">{props.title}</h2>
+      <span
+        className={`inline-block py-2 px-2 uppercase font-body text-xxs text-white font-bold rounded ${
+          "bg-" + tagColours[props.tag] || "bg-gray-experiment"
+        }`}
```

Comment:
The or `||` statements here don't ever evaluate to the second case. I tried creating a manual experiment here (without passing in a tag), and got `border-undefined` and `bg-undefined`.

Reply:
forgot to add brackets. Thanks for the catch